esmvaltool/recipes/recipe_validation.yml

```
@@ -48,7 +48,7 @@ diagnostics:
       rsut: # TOA SW up all sky
         preprocessor: pp_rad
     additional_datasets:
-      - {dataset: CERES-EBAF,  project: obs4MIPs, mip: Amon, level: L3B,  version: Ed2-7,  start_year: 2001,  end_year: 2012, tier: 1}
+      - {dataset: CERES-EBAF,  project: obs4MIPs, mip: Amon, level: L3B,  version: v20160610,  start_year: 2001,  end_year: 2012, tier: 1}
```

Wouldn't it be simpler for maintenance purposes to remove the `version` of `obs4MIPs` datasets in cases like that? Otherwise, we would have to update the recipes every time an obs4MIPs dataset is retracted from ESGF. 
```suggestion
      - {dataset: CERES-EBAF,  project: obs4MIPs, mip: Amon, level: L3B,  start_year: 2001,  end_year: 2012, tier: 1}
```

---

sounds good to me, man! I thought we ought to give it all the times, but really, I don't have obs4mips OCD :grin:

---

@remi-kazeroni In the future, I would propose to specify the `version` as much as possible for all datasets in all recipes. If a dataset is no longer publicly available the recipe is [broken according to our policy](https://docs.esmvaltool.org/en/latest/community/broken_recipe_policy.html) because people cannot obtain the data and it should be dealt with according to that policy.